docs/plugins/web-index.md

```diff
@@ -84,3 +84,26 @@ A custom template has the following macros available:
 | `$keywords`    | Define where the keywords meta tags will be injected into the html document         |
 | `$pre`         | Define where the prefetch/preload link tags will be injected into the html document |
 | `$title`       | Html Title                                                                          |
+
```

Comment:
You forgot to mention required options for the example as in engine: pug

Also a use of locals would be great too!
And thanks😃